.github/workflows/post-merge.yml

```diff
@@ -35,8 +35,13 @@ jobs:
 
           cd src/ci/citool
 
-          echo "Post-merge analysis result" > output.log
+          printf "*This is an experimental post-merge analysis report. You can ignore it.*\n\n" > output.log
+          printf "<details>\n<summary>Post-merge report</summary>\n\n" >> output.log
+
           cargo run --release post-merge-report ${PARENT_COMMIT} ${{ github.sha }} >> output.log
+
+          printf "</details>\n" >> output.log
```

Comment:
```suggestion
          printf "\n</details>\n" >> output.log
```

Reply:
Is the additional newline needed? The program output should end with a newline.

Reply:
I think it's better to always an empty line between the Markdown and HTML, but you already did it after the `<summary>` so maybe it's good enough?

Reply:
GitHub seems to accept it fine. So feel free to ignore the suggestion.